tableau-connector/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if test -z "$BASH_SOURCE"
+then
+      ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE:-$0}")" && pwd)/$(basename "${BASH_SOURCE:-$0}")"
+else
+      ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+fi
+CURRENT_FOLDER="${ABSOLUTE_PATH%/*}"
+echo "ABSOLUTEPATH=${ABSOLUTE_PATH}"
+echo "CURRENT_FOLDER=${CURRENT_FOLDER}"
+TARGET_FOLDER="$CURRENT_FOLDER"/target
+echo TARGET_FOLDER=${TARGET_FOLDER}
+mkdir -p $TARGET_FOLDER
+
+echo "Building Docker Image"
+docker build -t taco-builder $CURRENT_FOLDER
```

Comment:
building two times make you lost the label of the previous image. need or to check if the image exists or a mechanism of versioning. create a version and then create a tag latest, or if there is a image delete before create a new one, or if there is a image to not delete